package.json

```diff
@@ -60,7 +62,7 @@
     "isomorphic-fetch": "2.2.1",
     "jest": "^20.0.4",
     "lint-staged": "2.0.2",
-    "minifyify": "7.3.3",
+    "minify-stream": "^1.1.0",
```

Comment:
`minify-stream` wraps uglify-js in a stream interface, I'm using it to replace `minifyify` (which is unmaintained)